specs/data_structures.md

```diff
@@ -301,7 +303,7 @@ else
   node.n_max = max(l.n_max, r.n_max)
```

Comment:
I think the code snippet would be clearer if it mention that only the min namespace IDs are checked because if the min is a parity share, then the max should be too, and why.

Reply:
I could just change it to comparing `n_max`?

Reply:
But then it wouldn't be clear with n_max is compared but not n_min?

Reply:
Clarified in 455581a.